src/iotjs_binding.cpp

```diff
@@ -116,13 +119,17 @@ JObject::~JObject() {
 
 
 JObject& JObject::Null() {
-  static JObject null(JVal::Null(), false);
-  return null;
+  if (null == NULL) {
+    null = new JObject(JVal::Null(), false);
```

Comment:
@akosthekiss Okay, if it is single point of entry and exit, then i totally agree with you, and I think I did as you said.  Is it correct??